sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -424,6 +431,16 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
     return transaction;
   }
 
+  private @NotNull SentryTransaction applyScope(
```

Comment:
Here very likely we should populate transactions with more data, i just don't want to put everything into single PR.

Reply:
true, we have an issue for that already